src/Tribe/REST/V1/Endpoints/Single_Attendee.php

```diff
@@ -92,4 +92,257 @@ public function get( WP_REST_Request $request ) {
+		$attendee_data[ 'attendee_source' ] = 'rest-api';
+		$validate_status = $this->validate_attendee_status( $attendee_data, $provider );
+
+		if ( is_wp_error( $validate_status ) ) {
+			return $validate_status;
+		}
+
+		/**
+		 * Filter REST API attendee data before creating an attendee.
+		 *
+		 * @since TBD
+		 *
+		 * @param array $attendee_data Attendee data.
+		 * @param WP_REST_Request $request Request object.
+		 */
+		$attendee_data = apply_filters( 'tribe_tickets_rest_api_post_attendee_data', $attendee_data, $request );
+
+		if ( is_wp_error( $attendee_data ) ) {
+			return $attendee_data;
+		}
+
+		return [
+			'ticket'   => $ticket_id,
+			'provider' => $provider,
+			'data'     => $attendee_data,
+		];
+	}
+
+	/**
+	 * Process Request data for updating an attendee.
+	 *
+	 * @param WP_REST_Request $request
+	 *
+	 * @return array|WP_Error
+	 */
+	public function prepare_update_attendee_data( WP_REST_Request $request ) {
+
+		$attendee_id = (int) $request->get_param( 'id' );
+		$found       = tribe_attendees()->by( 'id', $attendee_id )->found();
+
+		if ( ! $found ) {
+			return new WP_Error( 'invalid-attendee-id', __( 'Attendee ID is not valid.', 'event-tickets' ) );
+		}
+
+		$provider = tribe_tickets_get_ticket_provider( $attendee_id );
+
+		if ( ! $provider ) {
+			return new WP_Error( 'invalid-attendee-provider', __( 'Attendee provider not found.', 'event-tickets' ) );
+		}
+
+		$attendee        = $provider->get_attendee( $attendee_id );
+		$updated_data    = $request->get_params();
+		$validate_status = $this->validate_attendee_status( $updated_data, $provider );
+
+		if ( is_wp_error( $validate_status ) ) {
+			return $validate_status;
+		}
+
+		/**
+		 * Filter REST API attendee data before creating an attendee.
+		 *
+		 * @since TBD
+		 *
+		 * @param array $updated_data Data that needs to be updated.
+		 * @param WP_REST_Request $request Request object.
+		 * @param array $attendee_data Attendee data that will be updated.
+		 */
+		$attendee_data = apply_filters( 'tribe_tickets_rest_api_update_attendee_data', $updated_data, $request, $attendee );
+
+		if ( is_wp_error( $attendee_data ) ) {
+			return $attendee_data;
+		}
+
+		return [
+			'attendee_id' => $attendee_id,
+			'attendee'    => $attendee,
+			'data'        => $attendee_data,
+		];
+	}
+
+	/**
+	 * Validate Attendee status if available.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $data Attendee data.
+	 * @param Tribe__Tickets__Tickets $provider Provider for the selected ticket.
+	 *
+	 * @return array | WP_Error
+	 */
+	public function validate_attendee_status( $data, $provider ) {
+		if ( isset( $data['attendee_status'] ) ) {
+			$statuses = tribe( 'tickets.status' )->get_statuses_by_action( 'all', $provider );
+			if ( ! in_array( $data['attendee_status'], $statuses, true ) ) {
+				$error_message  = __( 'Supported statuses for this attendee are: ', 'event-tickets' );
+				$error_message .= implode( $statuses, ' | ' );
```

Comment:
```suggestion
				$error_message  = sprintf(
					// Translators: %s - List of valid statuses.
					__( 'Supported statuses for this attendee are: %s', 'event-tickets' ), 
					implode( $statuses, ' | ' )
				);
```

Reply:
I was trying to avoid the statuses from getting translated. As those are slugs and we should not allow translations on the statuses. Will this have the same effect?

Reply:
The statuses are not being translated in this case. Only the string with the `%s` in it. The note for translators is there to give them context.